Code/GraphMol/RGroupDecomposition/RGroupScore.h

```diff
@@ -67,10 +68,82 @@ struct CartesianProduct {
   }
 };
 
-RDKIT_RGROUPDECOMPOSITION_EXPORT double matchScore(
-    const std::vector<size_t> &permutation,
-    const std::vector<std::vector<RGroupMatch>> &matches,
-    const std::set<int> &labels);
+class RDKIT_RGROUPDECOMPOSITION_EXPORT RGroupScorer {
```

Comment:
Now that the scorer is an independent class, we might want to have some tests.

Reply:
I also think that not that this class has new functionality, it should be documented.